workload/src/Samsung.Tizen.Sdk/targets/Samsung.Tizen.Sdk.Packaging.targets

```
@@ -144,11 +146,28 @@ Copyright (c) Samsung All rights reserved.
                      Condition="Exists('$(TizenManifestFile)')">
       <Output TaskParameter="PackageName" PropertyName="TizenPackageName" />
       <Output TaskParameter="PackageVersion" PropertyName="TizenPackageVersion" />
+      <Output TaskParameter="ApiVersion" PropertyName="TizenApiVersion" />
       <Output TaskParameter="ApplicationList" ItemName="TizenPackageApplicationList" />
       <Output TaskParameter="FirstApplication" ItemName="TizenPackageFirstApplication" />
     </GetManifestInfo>
   </Target>
 
+  <!--
+  ===========================================================================
+  _CheckApiVersion
+  ===========================================================================
+  -->
+  <Target Name="_CheckApiVersion" DependsOnTargets="_TizenGetTpkInfoFromManifest">
+    <CheckApiVersion
+      ManifestApiVersion="$(TizenApiVersion)"
+      TargetFrameworkIdentifier="$(TargetFramework)"
```

I mean that we'd better use `TargetFrameworkIdentifier` property instead of `TargetFramework`. 
`TargetFrameworkIdentifier` will be `Tizen` if `TargetFramework` is `tizen40`, and it will be `NETCoreApp` if `TargetFramework` is `net6.0-xxxx`. 
```suggestion
      TargetFrameworkIdentifier="$(TargetFrameworkIdentifier)"
```

---

applied.